tests/extrusion/test_steady_advection_3D_extr.py

```diff
@@ -79,7 +79,8 @@ def test_right_to_left(mesh, DG0, W):
     velocity = Expression(("-1.0", "0.0", "0.0"))
     u0 = project(velocity, W)
 
-    inflowexpr = Expression("x[1] > 0.25 && x[1] < 0.75 ? 1.0 : 0.5")
+    # inflowexpr = Expression("x[1] > 0.25 && x[1] < 0.75 ? 1.0 : 0.5")
```

Comment:
Please don't leave old code in comments.

Reply:
Oops.